packages/frontend/src/pages/InferenceServerDetails.svelte

```diff
@@ -78,6 +78,33 @@ const generate = async (language: string, variant: string): Promise<void> => {
       "role": "user"
     }
   ]
+}`,
+        },
+      };
+      break;
+    case InferenceType.OPENVINO:
+      options = {
+        url: `http://localhost:${service?.connection.port || '??'}/v3/chat/completions`,
+        method: 'POST',
+        header: [
+          {
+            key: 'Content-Type',
+            value: 'application/json',
+          },
+        ],
+        body: {
+          mode: 'raw',
+          raw: `{
```

Comment:
suggestion: we could use `raw: JSON.stringify({ ... })` so we are sure the content is a valid JS object

Reply:
Not related to this PR and I guess you mean a valid JSON object but I guess it shoud be handled by the create snippet method ?